File: src/main/java/com/amm/kafkafirst/infrastructure/Message.java
package com.amm.kafkafirst.infrastructure;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;

public class Message {
    private int code;
    private String message;

    @JsonCreator
    public Message(@JsonProperty("code") int code, @JsonProperty("message") String message) {
        this.code = code;
        this.message = message;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        final StringBuffer sb = new StringBuffer("Message{");
        sb.append("code=").append(code);
        sb.append(", message='").append(message).append('\'');
        sb.append('}');
        return sb.toString();
    }
}

File: src/main/kotlin/com/amm/kafkafirst/infrastructure/kafka/Message.kt
package com.amm.kafkafirst.infrastructure.kafka

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

class Message @JsonCreator
constructor(@param:JsonProperty("code") var code: Int, @param:JsonProperty("message") var message: String?) {

    override fun toString(): String {
        val sb = StringBuffer("Message{")
        sb.append("code=").append(code)
        sb.append(", message='").append(message).append('\'')
        sb.append('}')
        return sb.toString()
    }
}